common.gypi

```diff
@@ -286,6 +286,14 @@
           'BUILDING_UV_SHARED=1',
         ],
       }],
+      [ 'OS=="linux"', {
+        'target_conditions': [
+          ['_type=="static_library"', {
+            'cflags': [ '-fPIC', ],
```

Comment:
I think instead of doing this, we should just default to `-fPIC`.  Done that in d4277d4e848cdfa98cf27a3166e52edeed375448.